js/main.js

```diff
@@ -70,5 +70,5 @@ $listingBack.addEventListener('click', goBackToHome);
 function goBackToHome(event) {
   $listingSection.className = 'listing-section hidden';
   $searchSection.className = 'search-section ';
-
+  window.location.reload();
```

Comment:
You should **_never_** use `location.reload`, it reloads the entire page, which is the most expensive single action you can take in a web app. Why are you doing this?